### PR TITLE
session: format some sql statements

### PR DESCRIPTION
**Pull request by bestgopher**
### What problem does this PR solve?

format some sql statements, sql keyword changed to uppercase。

### Release notes

- No Release Note

### CODE REVIEW

**Comment by bestgopher** [comment]
@zz-jason @SunRunAway 
Hi, I have opened a new pr base on [pr#16851](https://github.com/pingcap/tidb/pull/16851), because the older one conflicts with master.

File: session/bootstrap.go [hunk 1]
```diff
@@ -80,210 +80,211 @@ const (
...
-		original_sql text NOT NULL  ,
-      	bind_sql text NOT NULL ,
-      	default_db text  NOT NULL,
-		status text NOT NULL,
-		create_time timestamp(3) NOT NULL,
-		update_time timestamp(3) NOT NULL,
-		charset text NOT NULL,
-		collation text NOT NULL,
-		source varchar(10) NOT NULL default 'unknown',
+<<<<<<< HEAD
```
Comment (tangenta) [inline session/bootstrap.go:235]:
```suggestion

```

**Comment by bestgopher** [comment]
@tangenta Done!

**Review by zz-jason — APPROVED** [review]
LGTM

**Pull request merged**